app/src/components/formdemo/FormDemo.tsx

```diff
@@ -749,6 +750,35 @@ export default function FormDemo() {
 		),
 	});
 
+	const yuri_data = `{
```

Comment:
The object depicted in the string is actually a JavaScript object, not a JSON object. JSON objects _must_ be formatted using double quotes like this:

```json
{
  "author": "Yuri the Yeti",
  "gender": "Male",
  "age": "33",
  "hobbies": [
    "sledding",
    "ice fishing",
    "building snow man"
  ],
  "children": []
}
```

This distinction is important because JavaScript apps will automatically parse JSON objects as JavaScript objects, but JSON parsers are not able to parse a JavaScript object as JSON.